src/js/actions/search/commands.js

```diff
@@ -171,12 +250,23 @@ define(function (require, exports) {
      * 
      * @return {string}
     */
-    var _getSVGClass = function () {
+    var _getSVGClassMenuCommand = function () {
         return "menu-commands";
     };
+
+    /**
+     * Find SVG class for global shortcuts
+     * If this needs to vary based on the item, use category list as parameter 
+     * (see getSVGCallback type in search store)
+     * 
+     * @return {string}
+    */
+    var _getSVGClassGlobalShortcut = function () {
+        return "global-shortcuts";
```

Comment:
We are still deciding if Global shortcuts should be a separate section or not. If so, @placegraphichere would need to provide an icon.